tests/ui/const-generics/adt_const_params/alias_const_param_ty-1.rs

```diff
@@ -1,5 +1,7 @@
-//@ known-bug: #114456
+//@ check-pass
```

Comment:
This was fixed by removing the `eq` call as previously we did not normalize any aliases in the `ty` field before equating them.